tests/snapshots/execution_spec__test-scalars.md_merged.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: tests/execution_spec.rs
+expression: merged
+---
+schema @server(graphiql: true, hostname: "localhost", port: 8000) @upstream {
+  query: Query
+}
+
+input Email {
+
+}
+
+input PhoneNumber {
+
+}
+
```

Comment:
this should come as scalar

Reply:
it's the generated outpur from config.to_sdl().. I don't think it's related to this pr